SpEC/com_motion.py

```diff
@@ -208,10 +208,8 @@ def remove_avg_com_motion(path_to_waveform_h5='rhOverM_Asymptotic_GeometricUnits
     # Read the waveform data in
     w_m = read_from_h5(path_to_waveform_h5)
 
-    # Insert '_CoM' into the original `path_to_waveform_h5`; this will be the output path
-    path_to_new_waveform_h5 = path_to_waveform_h5.split('.h5', 1)
-    path_to_new_waveform_h5.insert(1, '_CoM.h5')
-    path_to_new_waveform_h5 = ''.join(path_to_new_waveform_h5).replace(w_m.descriptor_string + '_', '')
```

Comment:
**Explanation:**  The `descriptor_string` is automatically computed and prepended to the filename in `write_to_h5`, so it needs to be removed here in order to not have filenames that start with, e.g., `rhOverM_rhOverM_`.  This serves as a bit of a check that everything is working properly, because the output file will only be named correctly if the input file was correctly interpreted.  That's important because the transformation depends on the data type — for example, if the data is `h` or `Psi4`, if `r` is scaled out or not, and if `M` is scaled out or not.  So an unexpected filename is the big warning that something messed up, and should be very obvious to the user even if s/he ignored all the stdout while the code was running.